crates/modelardb_server/src/remote.rs

```diff
@@ -602,21 +625,10 @@ impl FlightService for FlightServiceHandler {
         &self,
         request: Request<Action>,
     ) -> StdResult<Response<Self::DoActionStream>, Status> {
-        let metadata = request.metadata().clone();
+        let request_metadata = request.metadata().clone();
```

Comment:
Unclear why the `MetadataMap` is cloned when the type of `validate_request`'s argument is `&MetadataMap` which [`Request.metadata()`](https://docs.rs/tonic/latest/tonic/struct.Request.html#method.metadata) return.